cluster-provision/gocli/docker/docker.go

```diff
@@ -14,16 +14,17 @@ import (
 	"time"
 
 	"github.com/docker/docker/api/types"
-	"github.com/docker/docker/api/types/container"
+	containertypes "github.com/docker/docker/api/types/container"
```

Comment:
nit: for the sake of readability - the other files don't rename this import, is there a conflict that requires this or can we drop the rename?